lib/src/common/variables.js

```diff
@@ -26,8 +29,9 @@ export const globalTokens = {
   hal_purple_s_38: "#5f249f",
   hal_purple_d_30: "#4b1c7d",
   hal_purple_d_20: "#321353",
-  hal_purple_d_70: "#9a6bb2",
-  color_purple_600: "#7d2fd0",
+  hal_purple_d_70: "#9A6BB2",
+  color_purple_600: "#7D2FD0",
```

Comment:
Please keep `#9a6bb2` and `#7d2fd0` in lowercase to keep it as in the rest of the components.

Reply:
done